_presenters/tim-allen.md

```diff
@@ -0,0 +1,24 @@
+---
+github: FlipperPA
+hidden: false
+layout: speaker-template
+mastodon: https://fosstodon.org/@FlipperPA
+name: Tim Allen
+permalink: /presenters/tim-allen/
+photo_url: /static/img/presenters/tallen.jpg
+slug: tim-allen
+twitter: FlipperPA
+website: https://PyPhilly.org/
+---
+
+Tim joined [Wharton Research Data Services](https://wrds-www.wharton.upenn.edu/)
+in 2008, after working for local companies and starting a few of his own. Having
+started programming at age six, he is currently an organizer of the Philadelphia
+Python Users Group, Code for Recovery, and a member of the Python Software Foundation
+and Django Software Foundation. Tim enjoys playing very loud guitar with lots of pedals,
+is an avid cyclist, was
+[the first person to sell real world items for virtual microcurrency](https://www.mtv.com/news/g3te5b/finally-you-can-buy-something-real-with-play-money)
+long before Bitcoin, and enjoys beating his head against brick walls (as demonstrated
+by his passion for coding, cycling in Philadelphia, and never-ending quest to get his
+cat to behave). A life-long Philadelphia resident, he graduated from the University
+of Pennsylvania in 1996 with an individualized major, “Computer Aided Information Acquisition”.
```

Comment:
@FlipperPA I grabbed your bio from last year. Let me know if we need any changes.

Reply:
Thank You!